src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC, useRef } from "react";
+import DropDownArrow from "../icons/DropDownArrow";
+import styles from "./Dropdown.module.css";
+
+interface Props {
+  options: Array<string>;
+  whenSelected: () => void;
```

Comment:
legge til className: det kan brukes for å sende inn ekstra styling direkte på komponenten dersom man trenger det i noen tilfeller

Reply:
done